pynecone/.templates/web/pcversion.txt

```diff
@@ -0,0 +1 @@
+0.1.11
```

Comment:
What the reason for using a txt file vs having a variable in the constants.py

Reply:
Oh wait nvm I get it because if they update pc the web folder could be made from the older one

Reply:
Yeah I tried the config way but this seemed easier